assembl/static2/css/components/timeline.scss

```diff
@@ -206,11 +206,10 @@
 
 @media screen and (max-width: $screen-xxs-max) {
   .timeline {
-    margin-top: 45px;
+    display: none;
```

Comment:
No timeline view on small mobile screens?

Reply:
yup, the display is shitty... we only kept it for desktop only.

Reply:
Man...this reeks of bad design on our part :(